src/fparser/two/tests/test_fortran2003.py

```diff
@@ -836,15 +823,6 @@ def test_structure_constructor():  # R457
     assert isinstance(obj, tcls), repr(obj)
     assert str(obj) == 't(s = 1, a)'
```

Comment:
As noted in #255, this is actually invalid due to C486: "The keyword= may be omitted from a component-spec only if the keyword= has been omitted from each preceding component-spec in the constructor." Please could you correct it.

Reply:
OK, I've updated it so that the second argument is also a keyword. Hopefully that is OK.